service/matching/physical_task_queue_manager_test.go

```diff
@@ -265,6 +315,10 @@ func TestLegacyDescribeTaskQueue(t *testing.T) {
 		tlm.backlogMgr.taskAckManager.addTask(startTaskID + i)
 	}
 
+	// Manually increasing the backlog counter since it does not get incremented by taskAckManager.addTask
+	// Only doing this for the purpose of this test
```

Comment:
nit: better for comments to be specific (while being concise), so phrases like "only doing this" are not great. No need to change though if you don;t want to, I'm just taking this PR review as a moment to give my opinions on comment style (of which there are many opinions, it's good to follow existing thoughts and guidelines in this space [see: https://carlomilanesi.wordpress.com/2012/02/09/grammar-style-for-code-comments/])

Reply:
I see, thanks for the pointer - I wanna keep this for now only because I had seen in other areas of the code base where we caution the reader to never do this manually

Let me know if this works!